02.calendar/cal.rb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+require "optparse"
+require "date"
+
+opt = OptionParser.new
+option = {}
+opt.on( "-m ARG", desc = "set month variable" ){ |m| option[:m] = m }
+opt.on( "-y ARG", desc = "set year variable" ){ |y| option[:y] = y }
+opt.parse(ARGV)
+
+option[:m] ? month = option[:m].to_i : month = Date.today.month
```

Comment:
```suggestion
month = option[:m] ? option[:m].to_i : Date.today.month
```
こっちの書き方が主流です。
`a ? b : c`の`b`や`c`で代入することは滅多にないです〜。